### PR TITLE
Fix Rename JS-TS Identifier Test

### PR DESCRIPTION
**Pull request by vrubezhny**
Fix Rename JS-TS Identifier Test

Signed-off-by: Victor Rubezhny <vrubezhny@redhat.com>

### CODE REVIEW

File: org.eclipse.wildwebdeveloper/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -8,7 +8,7 @@ Bundle-Activator: org.eclipse.wildwebdeveloper.Activator
 Bundle-Vendor: Eclipse Wild Web Developer project
 Require-Bundle: org.eclipse.ui,
  org.eclipse.core.runtime,
- org.eclipse.lsp4e;bundle-version="0.13.6",
+ org.eclipse.lsp4e;bundle-version="0.13.8",
```
Comment (mickaelistria) [inline org.eclipse.wildwebdeveloper/META-INF/MANIFEST.MF:11]: Why is this necessary? If not, let's just leave it as it since 0.13.8 is not in SimRel and causes the build to fail.

Comment (vrubezhny) [reply org.eclipse.wildwebdeveloper/META-INF/MANIFEST.MF]: Sorry. This is a trace left from using a locally built version of LSP4E. No need to be here. Fixed

**Comment by mickaelistria** [comment]
Thanks!
